src/nile/cli.py

```diff
@@ -88,26 +95,37 @@ def run(path, network):
 @click.argument("arguments", nargs=-1)
```

Comment:
A small suggestion: wouldn't be good to organize the decorators of click commands (by size or type?). Params of the method don't need to match the order of the decorator because Click's wrapper is the one passing the argument by name.

Reply:
Haha yeah, I agree it'd be a good idea to better organize the decorators. I was matching the order because it seemed practical, but I think it looks a little weird—especially with the different `@_option`s.